cognite/extractorutils/configtools/elements.py

```diff
@@ -220,6 +236,7 @@ class CogniteConfig:
     data_set_external_id: Optional[str]
     extraction_pipeline: Optional[EitherIdConfig]
     timeout: TimeIntervalConfig = TimeIntervalConfig("30s")
+    global_config: GlobalConfig = field(default_factory=GlobalConfig)
```

Comment:
While it's called `global_config` in the SDK, I'm not so sure about exposing that to users who don't necessarily even know what kind of software the extractor is. Perhaps something like `connection`?